docsteady/utils.py

```diff
@@ -145,6 +145,9 @@ def download_and_rewrite_images(value):
     soup = BeautifulSoup(value.encode("utf-8"), "html.parser", from_encoding="utf-8")
     rest_location = urljoin(Config.JIRA_INSTANCE, "rest")
     for img in soup.find_all("img"):
+        # print(img)
+        img_style = urljoin(rest_location, img["style"])
+        img_width = re.sub('[^0-9]','', img_style)
```

Comment:
Is there a reason why these two lines can't just be this:
`img_width = re.sub('[^0-9]', '', img["style"])`

it seems like that should be more robust